samples/FluentMigrator.Example.Migrator/FluentMigrator.Example.Migrator.csproj

```diff
@@ -25,5 +25,9 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\FluentMigrator.Runner\FluentMigrator.Runner.csproj" />
     <ProjectReference Include="..\FluentMigrator.Example.Migrations\FluentMigrator.Example.Migrations.csproj" />
+    <ProjectReference Include="..\..\src\FluentMigrator.Runner.SQLite\FluentMigrator.Runner.SQLite.csproj" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
I'm not sure what you are trying to do here with the example project?

Reply:
To clarify, the example Migrator csproj demonstrates the ideal future end-state for FLuentMigrator that we're inching towards in v4.  We ran into various technical challenges in v3, mostly due to under-documented APIs and design decisions in .NET Core SDK:

* It wasn't clear from a project management perspective whether .netstandard was like a framework or merely an API specification
* It wasn't clear how to run old assemblies on newer runtimes (See #1139 )
* There is magic with how `.deps.json` determines what assemblies get loaded into the default AssemblyLoadContext.  See #1016 - Using an in-process runner pretty much guarantees a correct `.deps.json` file gets generated.